tools/ldb_cmd.cc

```diff
@@ -317,6 +318,12 @@ LDBCommand::LDBCommand(const std::map<std::string, std::string>& options,
     column_family_name_ = kDefaultColumnFamilyName;
   }
 
+  itr = options.find(ARG_SECONDARY_PATH);
+  secondary_path_ = "";
```

Comment:
We do not need to assign "" to `secondary_path`.

Reply:
I just put it there to be symmetric:)